src/client.rs

```diff
@@ -106,39 +165,38 @@ fn apply_message(
         stats.bytes += end_pos;
     }
 
-    let Some(tick) = apply_tick(&mut cursor, world)? else {
-        return Ok(());
-    };
-    if cursor.position() == end_pos {
-        return Ok(());
-    }
+    let replicon_tick = bincode::deserialize_from(&mut cursor)?;
+    trace!("applying init message for {replicon_tick:?}");
+    *world.resource_mut::<RepliconTick>() = replicon_tick;
```

Comment:
Yes, I removed it on purpose. We never send empty messages anymore. Events (and updates now) 'snap' to the last replicon tick with changes.

Reply:
It will panic later anyway, the panic message won't be good, but maybe it's fine?
If not, how about `assert_eq`? This way the check will be at least "cold".
We can also do `debug_assert_eq`.

Please, suggest, I don't have a preference.